docs/sources/writing-guide/front-matter/index.md

```diff
@@ -82,42 +92,73 @@ For `labels.products`, the supported values and the resulting published labels a
 - `enterprise`: "Enterprise"
 - `oss`: "Open source"
 
-Labels can be inherited through cascading front matter.
-Each project has a set of default labels that are defined in the root `_index.md` file of the project.
+Labels are inherited through cascading front matter.
+All labels, including project defaults, are set in the project's root index file.
+This is almost always the `docs/sources/_index.md` file in the project repository.
 
-For versioned projects, the `_index.md` file resides in the `website` repository.
-For unversioned projects, the `_index.md` file resides in the project’s repository.
+If you add a new page, consider whether the default labels are appropriate.
+If the default labels are incorrect for a page or directory of pages, update the cascading front matter in `docs/sources/_index.md`.
 
-If the default labels are incorrect for a page or directory of pages, update the labels.
-Also, if you are adding a new page, consider whether the default labels are appropriate.
-For each page, include a label in the `labels.products` sequence for every product that the page relates to.
+#### Set cascading front matter
 
-For example, if a **single page** describes a feature available in Grafana Cloud and Grafana Enterprise, the source file front matter should include the following:
+Set cascading front matter by adding or updating the `cascade` mapping in the `docs/sources/_index.md` front matter:
 
 ```yaml
-labels:
-  products:
-    - cloud
-    - enterprise
+cascade: <TARGETS...>
 ```
 
-For a **directory of pages** that describe a feature only available in Grafana Cloud, the branch bundle `_index.md` file front matter should include the following:
+Where `<TARGETS...>` is a sequence of mappings that looks similar to the following:
 
 ```yaml
-cascade:
+- _target: { path: <PATH> }
+  labels:
+    products: <PRODUCTS...>
+```
+
+- `<PATH>` is a glob pattern that matches the affected source pages.
+  For more information about the the `<PATH>` value, refer to [Specify a path](#specify-a-path).
+
+- `<PRODUCTS...>` is a sequence of valid product labels.
+
+The following example target mapping sets the product label to "Grafana Cloud" for all pages under `/docs/grafana/latest/troubleshooting/`:
+
+```yaml
+- _target: { path: /docs/grafana/*/troubleshooting/** }
   labels:
     products:
       - cloud
 ```
 
+##### Specify a path
+
+The value of the `_target.path` mapping is a string matching paths from the Hugo content root.
+Glob patterns specified with `*` and `**` match multiple paths.
+
+The `*` pattern matches any file or directory.
+It is useful for matching any version directory for versioned documentation.
+
+The `**` pattern matches all directories and files recursively.
```

Comment:
I was about to ask for an example in this section and then saw the section above. Thinking we could roll this into the previous section where you end up describing a path anyway.